### PR TITLE
Fixed aspect ratio in image preview, removed text under polaroid

### PR DESCRIPTION
**Pull request by JordanTheriault**

### CODE REVIEW

File: src/extensions/default/ImageUrlCodeHints/selfieWidget.html [hunk 1]
```diff
@@ -7,10 +7,8 @@
 	</div>
 	<div id="selfie-gallery">
 	    <div class="polaroid">
-	    	<p>Your new selfie</p>
 	    	<img id="selfie-photo" /><br>
 		</div>
```
Comment (humphd) [inline src/extensions/default/ImageUrlCodeHints/selfieWidget.html]: Fix the indent on this closing `div`

File: src/extensions/default/ImageUrlCodeHints/selfieWidget.html [hunk 2]
```diff
@@ -7,10 +7,8 @@
 	</div>
 	<div id="selfie-gallery">
 	    <div class="polaroid">
-	    	<p>Your new selfie</p>
 	    	<img id="selfie-photo" /><br>
```
Comment (humphd) [inline src/extensions/default/ImageUrlCodeHints/selfieWidget.html]: Do you need a `br` at the end here?

File: src/extensions/default/ImageUrlCodeHints/style.less [hunk 3]
```diff
@@ -11,6 +11,11 @@
   font-weight: bold;
 }
 
+.modal-body {
+  max-height: 290px;
+  overflow-y:hidden;
```
Comment (humphd) [inline src/extensions/default/ImageUrlCodeHints/style.less]: Space after `:`

**Comment by humphd** [comment]
A couple style nits, r+ with those.

File: src/extensions/default/ImageUrlCodeHints/selfieWidget.html (hunk 2 above)
Comment (JordanTheriault) [inline src/extensions/default/ImageUrlCodeHints/selfieWidget.html]: Good catch, no longer since I removed the caption.

**Pull request merged**